src/parser.js

```diff
@@ -0,0 +1,59 @@
+require('dotenv').config();
+const bent = require('bent');
+const jsdom = require('jsdom');
+
+const request = bent('string');
+const { JSDOM } = jsdom;
+
+/*
+* getData() returns Promise { <pending> }
+* It gets the data from 'pre' tag in a provided url
+* Splits the data into lines so its easier to process as a string array
+* That data is then returned as a Promise
+*/
+function getData() {
+  const list = request(process.env.FEED_URL)
```

Comment:
You can just do `return request(process.env.FEED_URL)` here, no need for `const list`.